package.json

```diff
@@ -110,6 +110,7 @@
     "why-did-you-update": "^1.0.6"
   },
   "dependencies": {
+    "@babel/runtime": "^7.4.5",
```

Comment:
This should also be a peer dep as well as a dep (oops, v3.3 should have been a v4 because of the peer dep requirement)

Reply:
Why should it be a peerDep?

Reply:
i suppose it doesn't have to me; it'd ensure maximal deduping, but since it's not stateful i guess we can rely on npm for that.

Reply:
Cool, that's what I thought as well but peerDeps are always a bit confusing to me so I wanted to make sure.